haystack/document_stores/types/protocol.py

```diff
@@ -1,23 +1,16 @@
 from typing import Protocol, Optional, Dict, Any, List
 import logging
-from enum import Enum
 
 from haystack.dataclasses import Document
 
+from .policy import DuplicatePolicy
```

Comment:
```suggestion
from haystack.document_stores.types.policy import DuplicatePolicy
```

Reply:
I was actually thinking... why not switch to relative import, only for siblings? This would be consistent with modern Python style, see https://docs.astral.sh/ruff/rules/relative-imports/

Reply:
Deal, let's do this later and not as-we-go 👍